### PR TITLE
command for reindexing reporting + trigger reindex on device deployments history reset 

### PR DESCRIPTION
**Pull request by kjaskiewiczz**
Tickets:
[MEN-6252](https://tracker.mender.io/browse/MEN-6252)
[MEN-6253](https://tracker.mender.io/browse/MEN-6253)

### CODE REVIEW

File: client/workflows/client.go [hunk 1]
```diff
@@ -238,3 +246,60 @@ func (c *client) StartReindexReportingDeployment(ctx context.Context,
...
+	info []DeviceDeploymentShortInfo) error {
+	if _, ok := ctx.Deadline(); !ok {
+		var cancel context.CancelFunc
+		ctx, cancel = context.WithTimeout(ctx, defaultTimeout)
+		defer cancel()
+	}
+	ident := identity.FromContext(ctx)
+	if ident == nil {
+		return errors.New("workflows: context lacking tenant identity")
+	}
```
Comment (tranchitella) [inline client/workflows/client.go:260]: This won't work, the context doesn't have the identity set and it fails:

```
$ docker exec -it integration-mender-deployments-1 deployments propagate-reporting
WARN[2023-02-02T10:39:59Z] 'presign.secret' not configured. Generating a random secret.  file=config.go func=config.Setup line=229
INFO[2023-02-02T10:39:59Z] propagating deployments history for all tenants  file=main.go func=main.selectDbs line=252
INFO[2023-02-02T10:39:59Z] no tenant DBs found - will try the default database deployment_service  file=main.go func=main.selectDbs line=261
INFO[2023-02-02T10:39:59Z] propagating deployments data to reporting from DB: deployment_service  file=main.go func=main.tryPropagateReportingForDb line=279
INFO[2023-02-02T10:39:59Z] Done with DB deployment_service, but there were errors: workflows: context lacking tenant identity.  file=main.go func=main.tryPropagateReportingForDb line=292
ERRO[2023-02-02T10:39:59Z] giving up on DB deployment_service due to fatal error: workflows: context lacking tenant identity  file=main.go func=main.propagateReporting line=233
INFO[2023-02-02T10:39:59Z] all DBs processed, exiting.                   file=main.go func=main.propagateReporting line=238
workflows: context lacking tenant identity
```

Either you pass in the tenant ID, or you populate the context's identity before calling this function. Tested with OS locally.

**Review by tranchitella — CHANGES_REQUESTED** [review]

**Review by tranchitella — APPROVED** [review]
Works fine.

**Comment by kjaskiewiczz** [comment]
@mender-test-bot start pipeline

**Comment by kjaskiewiczz** [comment]
basic pipeline - https://gitlab.com/Northern.tech/Mender/deployments/-/pipelines/765379504
don't know why it's not linked to the PR...

**Comment by kjaskiewiczz** [comment]
@mender-test-bot start pipeline

**Pull request merged**